### PR TITLE
[REF] Fine-grained logging (make FMRIPREP less shy)

### PR DESCRIPTION
**Pull request by oesteban**
Enables the `-v` flag for fine-grained tunning of FMRIPREP's quietness.
The default level is 25, only IMPORTANT messages or above (WARNING, ERROR)
will be written.
With one `-v` the level goes to 20 (INFO). Two (`-vv`) maps to the VERBOSE
new level. Finally, three (`-vvv`) is the DEBUG level.

### CODE REVIEW

**Comment by effigies** [comment]
Changing the `-v` flag is going to need to happen in `fmriprep-docker`, as well.

**Comment by oesteban** [comment]
:+1: will do

File: wrapper/fmriprep_docker.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def get_parser():
 
     parser.add_argument('-h', '--help', action='store_true',
                         help="show this help message and exit")
-    parser.add_argument('-v', '--version', action='store_true',
+    parser.add_argument('--version', action='store_true',
```
Comment (effigies) [inline wrapper/fmriprep_docker.py]: Also, line 156, in `expected_overlap`, `'v'` should become `'version'`.

**Comment by oesteban** [comment]
@effigies could you double check that my last commit makes sense?

**Comment by effigies** [comment]
It looks reasonable. Have you built an image and checked that `--help` and `--version` work? (Using the `-i` flag to specify the image should help with the weirdness of using it from the source directory.)

**Comment by oesteban** [comment]
They work. Merging.

**Pull request merged**